### PR TITLE
Added check for center what to common and a case to observe.

### PR DESCRIPTION
**Pull request by jayRyan24**
To fix the issue #3923:
```
R> 
[astrology]>get my telescope

[walkingastro]>stow my telescope

You get a maple simple telescope from inside your duffel bag.
> 
You put your telescope in your duffel bag.
> 
[astrology]>center telescope on Dawgolesh

Center what?
```

### CODE REVIEW

File: astrology.lic [hunk 1]
```diff
@@ -279,6 +279,9 @@ class Astrology
 
     if @have_telescope
       case DRCMM.center_telescope(body)
+      when /Center what?/
```
Comment (rpherbig) [inline astrology.lic]: ❓ Does this work without a 15 second delay? I don't see `Center what?` in the list of matching messages in `DRCMM.center_telescope`

File: common-moonmage.lic [hunk 2]
```diff
@@ -54,7 +54,7 @@ module DRCMM
   end
 
   def center_telescope(target)
-    DRC.bput("center telescope on #{target}", 'You put your eye', 'open it to make any use of it', 'The pain is too much', "That's a bit tough to do when you can't see the sky", 'Your search for')
+    DRC.bput("center telescope on #{target}", 'Center what?', 'You put your eye', 'open it to make any use of it', 'The pain is too much', "That's a bit tough to do when you can't see the sky", 'Your search for')
```
Comment (jayRyan24) [inline common-moonmage.lic]: Center what?

Comment (jayRyan24) [reply common-moonmage.lic]: @rpherbig

File: astrology.lic (hunk 1 above)
Comment (jayRyan24) [reply astrology.lic]: Should be down below.

File: common-moonmage.lic (hunk 2 above)
Comment (rpherbig) [reply common-moonmage.lic]: Did you test that this works?

For some reason I have it in my head that `bput` interprets the match message as a regex, so the `?` in the message gives me pause. Will it be interpreted as the literal `?` or as "Match between 0 and 1 of the preceding token."?

Comment (jayRyan24) [reply common-moonmage.lic]: I just removed it, no reason to include it anyway.

Comment (jayRyan24) [reply common-moonmage.lic]: Good call.

**Pull request merged**